utils/writeable/src/lib.rs

```diff
@@ -81,6 +81,14 @@ use core::fmt;
 
 pub use try_writeable::TryWriteable;
 
+/// Helper types for trait impls.
+pub mod adapters {
```

Comment:
Note @robertbastian: this is where we could put the `LossyWriteable` wrapper if needed.